### ISSUE TITLE
Introduire les alias des commandes npm ?

### ISSUE THREAD

**Issue by thom4parisot** [oncletom/nodebook#302]
Sous forme d'un tableau récapitulatif, dans chaque section (en parenthèses).
Je ne sais pas trop :-D

**Comment by oncletom** [comment]
Oui, dans un aparté. Trouver un endroit qui s'y prête. Dans un endroit qui récapitule ?
